spec.html

```diff
@@ -4409,7 +4409,7 @@ <h1>The Abstract Closure Specification Type</h1>
 
     <emu-clause id="sec-data-blocks">
       <h1>Data Blocks</h1>
-      <p>The <dfn>Data Block</dfn> specification type is used to describe a distinct and mutable sequence of byte-sized (8 bit) numeric values. A Data Block value is created with a fixed number of bytes that each have the initial value 0.</p>
+      <p>The <dfn>Data Block</dfn> specification type is used to describe a distinct and mutable sequence of byte-sized (8 bit) numeric values. A <em>byte</em> is an integer value in the range 0 through 255, inclusive. A Data Block value is created with a fixed number of bytes that each have the initial value 0.</p>
```

Comment:
```suggestion
      <p>The <dfn>Data Block</dfn> specification type is used to describe a distinct and mutable sequence of byte-sized (8 bit) numeric values. A <dfn>byte</dfn> is an integer value in the range 0 through 255, inclusive. A Data Block value is created with a fixed number of bytes that each have the initial value 0.</p>
```

so it auto-links?

Reply:
I didn't want the auto-linking, it seems excessive. My thought was byte could be used in many different contexts where the auto-linking has dubious value.

Reply:
what about if you call it a "byte value" instead of just a "byte"? then the few times it's used, it would usefully link, but prose discussion of bytes wouldn't be noisy.